templates/about.html

```diff
@@ -10,6 +10,15 @@ <h2> Welcome! </h2>
 Little is known about plant movement. More is known about gravitropism, the response of plants to gravity. Less is known about circumnutation, the continuous waving motion in plant parts (leaves, flowers, shoots) that was described by Charles Darwin over 100 years ago. Identification of mutant Arabidopsis (the genetic model plant) lines that are abnormal in plant movement sets the stage to understand the mechanics of plant movement.
 </p>
 
+<p class='demo'>
+  You are currently logged in as the Plant Tracer Demo user. You have
+  limited functionality in this mode. For full functionality, please
+  log in as a full user.
+</p>
+<p class='nodemo'>
+  You are not in demo mode.
+</p>
+
```

Comment:
@sbarber2 - For example, please check out this.

Reply:
got it, thanks